db/models.py

```diff
@@ -50,3 +57,54 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(on_delete=models.CASCADE, to="User")
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ["-created_at"]
```

Comment:
class Meta should be placed before methods. Don't forget to read checklist, please

Reply:
Fix everywhere